examples/identity_py/eval.py

```diff
@@ -0,0 +1,93 @@
+import logging
+import time
+from logging import getLogger, FileHandler, StreamHandler
+import os
+import sys
+import math
+
+from utils import set_seed
+from submission import custom_kernel
+from reference import ref_kernel, check_implementation, generate_input
+
+WARMUP_RUNS = 10
+TIMED_RUNS = 100
+
+logger = getLogger("PopcornOutput")
```

Comment:
Is logger here actually the right abstraction. This is less logging, and more actual program output that just  gets redirected to something else than stdout/stderr.